### DISCUSSION TITLE
Nerdctl windows server CNI not working (`incompatible CNI versions; config is "1.0.0", plugin supports ["0.2.0"]`)

### DISCUSSION THREAD

**Discussion by AristotelisNikolas** [containerd/nerdctl#3747] [Q&A] [answered]
Good Evening, i would like to ask a detailed guide installed for windows server and a proper CNI that works on Windows Server.

So far i managed to install nerdctl add a nat network pull an image try to run it but it keeps have problems with the cni network,
i have on env path the directory with the bin folder.
![cni version](https://github.com/user-attachments/assets/3c633ab4-adda-4ecd-9fd0-1e3902f44062)
![cni version 2](https://github.com/user-attachments/assets/42c52a1c-c389-45c3-a300-b2a8bf711743)

I would like to ask if there is a solution for windows that the cni works and if it does can you share a detailed guide to installed.

**Comment by AkihiroSuda** [comment]
CNI plugins seems extremely old (6/5/2019)

**Reply by AristotelisNikolas** [reply]
Good evening, you mean this one [https://github.com/containernetworking/plugins/releases/](url) i have try also too but it has no nat.exe and the other exe doesn't work at all see tha picture below.
![no nat cni version](https://github.com/user-attachments/assets/b71f9507-d571-460f-aa4d-6786d670c064)

**Reply by AkihiroSuda** [reply]
Can you try https://github.com/Microsoft/windows-container-networking

**Reply by AristotelisNikolas** [reply, marked as answer]
Thank you very much it worked
![it works](https://github.com/user-attachments/assets/107b7758-305d-44f6-97c2-c347bc873f05)
👍
